index.bs

```diff
@@ -375,8 +383,8 @@ spec: webidl
         <code>{name: {{"midi"}}, sysex: true}</code> ("midi-with-sysex") is
         <a>stronger than</a> <code>{name: {{"midi"}}, sysex: false}</code>
         ("midi-without-sysex"), so if the user denies access to
-        midi-without-sysex, the UA must also deny access to midi-with-sysex, and
-        similarly if the user grants access to midi-with-sysex, the UA must also
+        midi-without-sysex, the UA would also deny access to midi-with-sysex, and
```

Comment:
I think there's actually an RFC 2119 "must" here, although being in an example that's derived from other rules in the spec rather than being required here.